deploy/db/migrations/20170818120003_InitialSchema.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+)
+
+// Up is executed when this migration is applied
+func Up_20170818120003(txn *sql.Tx) {
+
+	databaseProvider := os.Getenv("DATABASE_PROVIDER")
+	fmt.Printf("ENV is: %s, %s", databaseProvider, os.Getenv("MYSQL_ROOT_PASSWORD"))
+	binaryDataType := "BYTEA"
+	if databaseProvider == "mysql" {
+		binaryDataType = "BLOB"
+	}
+	createTokens := "CREATE TABLE IF NOT EXISTS tokens ("
+	createTokens += "user_guid     VARCHAR(36) NOT NULL, "
+	createTokens += "cnsi_guid     VARCHAR(36), "
+	createTokens += "token_type    VARCHAR(4)  NOT NULL, "
+	createTokens += "auth_token    " + binaryDataType + "       NOT NULL, "
+	createTokens += "refresh_token " + binaryDataType + "       NOT NULL, "
+	createTokens += "token_expiry  BIGINT      NOT NULL, "
+	createTokens += "last_updated  TIMESTAMP NOT NULL DEFAULT (NOW()) )"
+
+	if databaseProvider == "pgsql" {
+		createTokens += " WITH (OIDS=FALSE);"
+	}
+
+	_, err := txn.Exec(createTokens)
+	if err != nil {
+		fmt.Printf("Failed ot migrate due to: %v", err)
```

Comment:
typo "ot"

Reply:
The function has no return params - how do we report that the migration failed - all we are doing is printing an error.